README.md

```diff
@@ -47,11 +47,11 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo]()
+- [Live Demo](https://richie1988.github.io/SuperMario/)
 
 ## 🚀 video presentation <a name="video-presentation"></a>
 
-- [ video presentation ]()
+- [ video presentation ](https://www.loom.com/share/7560711542d5401c8523acdade5bd0b0?sid=99b37e01-0364-4cf0-b0b6-ae6c6d0d2831)
```

Comment:
- [ ] In your video presentation you forgot to show us a piece of code you are proud of, kindly include that :+1: